src/core/routes/routes.js

```diff
@@ -10,7 +10,7 @@ export default angular.module('core')
             })
             .state({
                 name: 'search',
-                url: '/search?:foo',
+                url: '/search?:query',
```

Comment:
no additional instructions just use '/search?query'

Reply:
Fixed, thank you